site/content/docs/contributing/development.md

```diff
@@ -140,6 +140,19 @@ inner markdown content. Usage: `{{</* securitygoose */>}} Notice markdown conten
 
 1. `readFile` -- is used to inline file contents. Usage: `{{%/* readFile "static/examples/ingress/contour/patch.json" */%}}`
 
+### Style Guide
+
+KIND documentation is primarily maintained by code contributors and project maintainers, and as
+such does not strictly obey the [Kubernetes style guide]. However, we ask that
+documentation improvements adhere to [content best practices] and avoid [documentation anti-patterns].
+
+Only use the following when referring to the project or command line:
+
+- `kind` in all lowercase with backticks.
+- "KIND" in all uppercase.
+
+Using "kind" without backticks, "Kind" with only one uppercase, or "KinD" with leading and trailing
+uppercase is not permitted.
```

Comment:
There are always exceptions, so "discouraged" might be a better choice than "not permitted". But not that critical.